install/kubernetes/cilium/templates/cilium-operator/poddisruptionbudget.yaml

```diff
@@ -0,0 +1,16 @@
+{{- if and .Values.operator.enabled .Values.operator.podDisruptionBudget.enabled }}
+apiVersion: policy/v1beta1
+kind: PodDisruptionBudget
+metadata:
+  name: cilium-operator
+  namespace: {{ .Release.Namespace }}
+  labels:
+    io.cilium/app: operator
+    name: cilium-operator
+spec:
+  maxUnavailable: {{ .Values.operator.maxUnavailable }}
```

Comment:
should it be `.Values.operator.podDisruptionBudget.maxUnavailable` instead ?

Reply:
thanks.I'm fixing

Reply:
fixed